src/Menu.tsx

```diff
@@ -13,90 +14,123 @@ import { useTheme } from './theme';
 /**
  * MENU
```

Comment:
An open question about organization of files with multiple subcomponents like this. Do we like separating by subcomponent with comments like this? Or organization by types/interfaces on top, then components, then styles?
If we do it by subcomponent section like currently, I think the interface should always be before the component. Adds for easier readability since the interface nicely sums up what is needs. This is just me being nitpicky and does not actually require any changes :)

Reply:
It's alphabetical order via prettier - can't really change it.

Reply:
In general, we may want to split very large components, same as we do with Popup.